test/integration/run-test.js

```diff
@@ -1,5 +1,7 @@
 require('module-alias/register');
 
+process.env.ASK_SHARE_USAGE = false;
```

Comment:
I dont see an example like this currently. We can add to npm test script. We would have to add to all tests scripts: test, test:report, etc. If we want it to work on windows we also need to add cross-env package. this way was easier, but I am open for ideas.

Reply:
Yea I think the order to solve this case is: mock metricClient > disable in scripts > manage it in the source code. Can we create a test runner and then make sure for any test the metricClient operations get stubbed?

Reply:
1)  for unit tests - potentially high number of test cases to update.There is chance to forget to mock for new tests unless we are doing it in one place globally. 
2) for integration and functional tests - we cannot stub since those tests run as child process.

for consistency and simplicity, I decided to use env variable.

Reply:
The only issue of doing this approach is: running unit test will pollute the existing env setting. Could you set ASK_SHARE_USAGE back to what it was before running tests? I guess we can use env var until we notice some real issue.

Reply:
I dont think it pollutes env settings. It is only in process memory of the test script.

```
echo $ASK_SHARE_USAGE

npm test
....

echo $ASK_SHARE_USAGE

```